libraries/engage/core/helpers/i18n.spec.js

```diff
@@ -0,0 +1,29 @@
+import { i18n } from './i18n';
+
+describe('i18n', () => {
+  const methods = ['text', 'price', 'date', 'time', 'number'];
+  methods.forEach((m) => {
```

Comment:
As i said, it is not safe to throw error before App is started. Better to remove it

Reply:
OK

Reply:
It will still throw an error so I guess it's ok to leave that test case.